be/src/util/mem_info.h

```diff
@@ -39,6 +41,20 @@ class MemInfo {
         return _s_physical_mem;
     }
 
+    static inline int64_t current_mem() {
+        size_t allocated_bytes = 0;
```

Comment:
This value should be updated in period, rather than get it every time we call `current_mem()`.
`GetNumericProperty()` seems not a very-slight function call.

https://github.com/gperftools/gperftools/blob/f7c6fb6c8e99d6b1b725e5994373bcd19ffdf8fd/src/tcmalloc.cc#L700